docs/extend/plugins_network.md

```diff
@@ -9,42 +9,39 @@ weight=-1
 +++
 <![end-metadata]-->
 
-# Docker network driver plugins
+# Docker Network Plugins
```

Comment:
note that we use [sentence capitalization](https://github.com/docker/docker/blob/master/docs/project/doc-style.md#in-headings) - not sure if "Docker Network Plugins" is a product name